Base/usr/share/man/man1/printf.md

```diff
@@ -0,0 +1,52 @@
+## Name
+
+printf - format and print data
+
+## Synopsis
+
+```**sh
+printf FORMAT [ARGUMENT]...
+```
+
+## Description
+
+`printf` formats _ARGUMENT_(s) according to _FORMAT_ and prints the result to standard output.
+
+_FORMAT_ is similar to the C printf format string, with the following differences:
+- The format specifier `b` (`%b`) is not supported
+- The format specifiers that require a writable pointer (e.g. `n`) are not supported
+- The format specifier `q` (`%q`) has a different behaviour, where it shall print a given string as a quoted string, which is safe to use in shell inputs.
```

Comment:
This item is a complete sentence, with a period at the end. Others should probably be, too.